catalog/gcpm041u-lemur/projects_gcpm041u-lemur_assets_scanfi_v12_SCANFI_v1_2.jsonnet

```diff
@@ -0,0 +1,322 @@
+local id = 'projects/gcpm041u-lemur/assets/scanfi_v12/SCANFI_v1_2';
+local subdir = 'gcpm041u-lemur';
+//local subdir = 'NRCan_CFS_LFC';
+local version = '1.2';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.ogl_canada_2_0;
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+  title: 'SCANFI: the Spatialized CAnadian National Forest Inventory data product' + version,
+  version: version,
+
+  description: |||
+    This data publication contains a set of 30m resolution raster files representing 2020 
+    Canadian wall-to-wall maps of broad land cover type, forest canopy height, degree of 
+    crown closure and aboveground tree biomass, along with species composition of several 
+    major tree species. The Spatialized CAnadian National Forest Inventory data product (SCANFI) 
+    was developed using the newly updated National Forest Inventory photo-plot dataset, which 
+    consists of a regular sample grid of photo-interpreted high-resolution imagery covering all 
+    of Canada’s non-arctic landmass. SCANFI was produced using temporally harmonized summer and 
+    winter Landsat spectral imagery along with hundreds of tile-level regional models based on 
+    a novel k-nearest neighbours and random forest imputation method.
+    
+    A full description of all methods and validation analyses can be found in [Guindon et al. (2024)](https://doi.org/10.1139/cjfr-2023-0118). 
+    As the Arctic ecozones are outside NFI’s covered areas, the vegetation attributes in these 
+    regions were predicted using a single random forest model. The vegetation attributes in these 
+    arctic areas could not be rigorously validated. The "articExtrapolationArea" band identifies these zones.
+    
+    Data limitations:
+    1- The spectral disturbances of some areas disturbed by pests are not comprehensively represented in the 
```

Comment:
if you add a newline above and change "1-" to "1. ", we will get a markdown list. But don't worry too much about this - we can fix the formatting internally.